### ISSUE TITLE
[Textarea] Change counter text color to .acce1

### ISSUE THREAD

**Issue by hugompfigueira** [lumada-design/hv-uikit-react#1286]
**Describe the bug**
The Textarea pattern has a text that doesn't pass the Accessibility criteria. The text should be changed to **.acce1** color in order to pass the contrast check.

**Expected behavior**
Have Textarea pattern info text changed to **.acce1** color.

**Screenshots**
<img width="640" alt="Screenshot 2020-03-18 at 18 37 47" src="https://user-images.githubusercontent.com/35467239/76995634-0cb56e00-6948-11ea-87a3-ad44b7c6fe52.png">

**Enviroment (please complete the following information):**

- Device: MacBook Pro 15
- OS: 10.14.6
- Browser: Chrome & Safari
- Version: 80
